scss/standalone/patterns_lists-combined.scss

```diff
@@ -0,0 +1,6 @@
+@import '../vanilla';
```

Comment:
Please update to use single standalone SCSS for all lists examples, as per forms.